### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by deepakhangingpanda**
create  MR

### CODE REVIEW

**Comment by deepakhangingpanda** [comment]
delete button added

**Comment by deepakhangingpanda** [comment]
Refactor: Files structor changes

File: App.js [hunk 1]
```diff
@@ -1,82 +1,114 @@
...
+
+  const closeRow = (rowMap, rowKey) => {
+    if (rowMap[rowKey]) {
+      rowMap[rowKey].closeRow();
+    }
+  };
+
+  const deleteRow = (rowMap, rowKey) => {
+    closeRow(rowMap, rowKey);
+    console.log({rowKey}, {rowMap});
```
Comment (hangingPandaReact) [inline App.js:42]: remove console

File: App.js [hunk 2]
```diff
@@ -1,82 +1,114 @@
...
+          <TextInput
+            style={styles.input}
+            placeholder={'Write a task'}
+            value={task}
+            onChangeText={text => setTask(text)}
+          />
+        ) : null}
+
+        <TouchableOpacity
+          onPress={() => {
```
Comment (hangingPandaReact) [inline App.js:103]: For this onPress we can create a function and then call it here

**Pull request merged**